x_obs: [0]
y_obs: [0]
